CMakeLists.txt

```diff
@@ -182,6 +182,14 @@ if( BUILD_CLIENTS_SAMPLES OR BUILD_CLIENTS_TESTS OR BUILD_CLIENTS_BENCHMARKS )
         DEB "${GFORTRAN_DEB}"
         RPM "${GFORTRAN_RPM}")
   endif()
+  if(BUILD_CLIENTS_BENCHMARKS)
```

Comment:
Should this be BUILD_CLIENTS_SAMPLES?

Reply:
Thanks for catching this, correction in next commit.